Header header
string joint_name
int32 motor_id
bool calibrated_flag
float64 position
float64 velocity
float64 torque
float32 temperature
int32 error_flags